runtime/Runtest/intrinsics.c

```diff
@@ -31,6 +32,23 @@ static unsigned char rand_byte(void) {
   return x & 0xFF;
 }
 
+static void report_internal_error(const char *msg, ...)
```

Comment:
@delcypher Do klee_warning and klee_error lead to circular or weird dependences? Because this seems functionally similar to those methods.